cypress/integration/formWithAsync.spec.js

```diff
@@ -0,0 +1,157 @@
+beforeEach(() => {
+  cy.visit('http://localhost:3000')
+})
+
+describe('Check if FormWithAsync works', () => {
+  it('Testing input creation and deletion', () => {
+    cy.get('form[data-testid="formWithAsync"]').within(($form) => {
+      cy.get('button')
+        .contains('Add an Input')
+        .should('be.enabled')
+        .click()
+        .click()
```

Comment:
why twice click function invoked ?

Reply:
to test if creation of multiple inputs work